mithril-common/src/entities/snapshot.rs

```diff
@@ -19,6 +20,9 @@ pub struct Snapshot {
 
     /// Compression algorithm of the snapshot archive
     pub compression_algorithm: CompressionAlgorithm,
+
+    /// Version of the Cardano node used to product immutable files.
```

Comment:
This is probably more accurate
```suggestion
    /// Version of the Cardano node used to create snapshot archive.
```

Reply:
Yes, but the snapshot embeds other files than immutable files: these are the files that could be incompatible between two Cardano node versions, as detailed in #947. Otherwise, this information would not be very relevant...